org.civicrm.reqangular/src/common/vendor/angular/angular-animate.min.js.map

```diff
@@ -1,8 +0,0 @@
-{
```

Comment:
You delete map files, could you please explain why in the PR description?

Reply:
good catch, i didn't notice this at all, i don't know why it happened. will check

Reply:
It happens in the first commit (https://github.com/compucorp/civihr/pull/2772/commits/76e23a38b1a1a5313a66eb0058752abe47064a6c), at which point our [.gitignore](https://github.com/compucorp/civihr/blob/staging/.gitignore) kicks in and removes them
```
# Exclude all JS source map files
*.js.map
```
in any case we are going to (hopefully) switch to use npm soon for vendor files, so i think this can be ignore